src/pudl/extract/extractor.py

```diff
@@ -262,14 +262,19 @@ def extract(self, **partitions: PartitionSelection) -> dict[str, pd.DataFrame]:
         return all_page_dfs
 
 
-@op
+@op(tags={"memory-use": "high"})
```

Comment:
Only the EIA-930 concatenation is truly high memory (11GB!) but the other concatenations are extremely brief, so adding this concurrency limit to them incidentally should have very minor impacts on the overall DAG.